nixos/modules/virtualisation/build-vm.nix

```
@@ -31,6 +31,17 @@ in
 {
   options = {
 
+    virtualisation.isVmVariant = mkOption {
+      type = lib.types.bool;
+      default = false;
+      example = true;
+      description = ''
+        This option is `true` if the machine configurations was produced by `nixos-rebuild build-vm`.
+        Unlike {option}`virtualisation.vmVariant` this option can be used to specify inline and non-VM
```

Let's make this a bit more explicit.
```suggestion
        Unlike {option}`virtualisation.vmVariant`, where you can set definitions that are only for the VM, this option can be read from to make definitions conditional in the context being _not_ the VM variant.
```